client/my-sites/domains/domain-management/settings/dns/style.scss

```diff
@@ -0,0 +1,60 @@
+@import '@automattic/onboarding/styles/mixins';
+@import '@automattic/typography/styles/variables';
+@import '@wordpress/base-styles/breakpoints';
+
+.dns-card {
+  &__button {
+    margin-top: 16px;
+  }
+}
+
+.dns-record-item {
+	font-size: $font-body-small;
+	line-height: 1.7;
+	&.is-placeholder > div {
+		margin-bottom: 0.75em;
+		&::after {
+			animation: pulse-light 1.8s ease-in-out infinite;
+			background-color: var( --color-neutral-10 );
+			content: '';
+			display: block;
+			height: $font-body-small;
+		}
```

Comment:
There's a `placeholder` mixin that I believe you could use here instead of defining the animation and color yourself: `@include placeholder()` 🙂